README.md

```diff
@@ -86,6 +86,18 @@ Note: In this configuration, you must do some additional configuration:
 - If you wish your Plex Media Server to be accessible outside of your home network, you must manually setup port forwarding on your router to forward to the `ADVERTISE_IP` specified above.  By default you can forward port 32400, but if you choose to use a different external port, be sure you configure this in Plex Media Server's `Remote Access` settings.  With this type of docker networking, the Plex Media Server is essentially behind two routers and it cannot automatically setup port forwarding on its own.
 - (Plex Pass only) After the server has been set up, you should configure the `LAN Networks` preference to contain the network of your LAN.  This instructs the Plex Media Server to treat these IP addresses as part of your LAN when applying bandwidth controls.  The syntax is the same as the `ALLOWED_NETWORKS` below.  For example `192.168.1.0/24,172.16.0.0/16` will allow access to the entire `192.168.1.x` range and the `172.16.x.x` range.
 
+### Using `docker-compose` on ARM devices
+
+The provided `docker-compose` templates use the `plexinc/pms-docker` image which is the amd64 build and won't work on ARM devices.
```

Comment:
Does it try to use the amd64 image?  I would have expected that it would just fail to fetch because there isn't an image for the arm architecture.

Assuming that this is correct, maybe it's better to say that the arm images aren't currently available on dockerhub so you have to build it manually.

Reply:
When I tried it, yes, it built with the wrong arch.

Reply:
Huh, docker-compose built the (wrong architecture) image?  Was this when you previously had `build: .` in the file?  I'd be a bit surprised if it tries to build the image in its current form because that would mean it would have to find this repository in GH, clone it, and then build it.  Not entirely unreasonable for it to be able to do this but not what I would have expected.

Reply:
No, I had taken the "bridge" docker-compose yaml over to my RasPi and tried to start it from that. The only edits I made were related to the env vars. It wouldn't start up, and when I checked I was getting a weird instruction error, which I soon realized was related to architecture. Didn't see any docs around it, but figured if I just built the armv7 myself and docker-compose using that it would be fine, and sure enough it was. That's why I created the PR.

Reply:
To be clear I manually copied the Dockerfile-armv7 to the Pi to run the build, nothing was automatically cloned from GH. Not entirely sure what you meant there, sorry.

Reply:
That sounds like it grabbed an image of the wrong architecture.  That's very disappointing because the images do have labels indicating their architecture so docker has no excuse doing this.  Oh well.

Reply:
Yeah, it was surprising, but I'm not a Docker expert, so I wasn't sure the reason behind it. I guess I kind of assumed Docker couldn't find the right architecture and defaulted to arm64?